llvm/test/CodeGen/AArch64/speculation-hardening.mir

```diff
@@ -11,34 +12,148 @@
 # - other direct branches don't seem to be generated by the AArch64 codegen
 --- |
   define void @nobranch_fallthrough(i32 %a, i32 %b) speculative_load_hardening {
+  ; CHECK-LABEL: nobranch_fallthrough:
```

Comment:
If you use the autogenerated test, make sure to clear the all check lines.
As is, there are labels that are repeated which may lead in weird failures with FileCheck. (e.g., the label nobranch_fallthrough appears twice.)